### PR TITLE
Image analysis

### PR DESCRIPTION
**Pull request by facumunoz**

### CODE REVIEW

**Review by InfinityGrip — APPROVED** [review]

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -23,8 +23,20 @@ <h1>My Portfolio</h1>
                 <input type="file" accept="image/*" name="image">
                 <button>Upload</button>
             </form>
+            <form action="/image-analysis" method="POST" enctype="multipart/form-data">
```
Comment (Corramel) [inline src/main/webapp/index.html:30]: There's two forms here. (Three technically) Is this intended?

Comment (facumunoz) [reply src/main/webapp/index.html:30]: No, I guess it happened at some point when pulling in changes or something. I made a form for testing the image analysis feature before I had any other changes you guys made to the index.html. But now that the image analysis is in the same servlet as the rest of the stuff, there is only a need for one of course.

Comment (Corramel) [reply src/main/webapp/index.html:30]: Cool, that can be removed. Just need to make a note.

**Review by Corramel — APPROVED** [review]
Looks good, a couple of things may need to be changed but we can look at it as we go. I'll also use this to actually try and use tags instead of the mock-up version that I have currently.

**Comment by matty377** [comment]
I fixed a couple conflicts with `main`. I know you had a few more changes to move off of the `v1` dependencies. Will that be in a followup PR?

**Pull request merged**